docs/general/getting_started.md

```diff
@@ -96,3 +96,12 @@ class PostsController < ApplicationController
 
 end
 ```
+
+Do note when using Rails 5 API mode (or when you're not using `ActionController::Base` as the base controller class), you'll need to include `ActionController::Serialization` in your `ApplicationController`
+
+```ruby
+class ApplicationController < ActionController::API
+  include ActionController::Serialization
```

Comment:
Thanks for this!

However, I don't believe this is still true.  In Rails 5 I see the [ActionController::API  uses the `:action_controller` load hook](https://github.com/rails/rails/blob/v5.0.0.beta2/actionpack/lib/action_controller/api.rb#L144), so [our railtie should include it there, as well](https://github.com/rails-api/active_model_serializers/blob/v0.10.0.rc4/lib/active_model_serializers/railtie.rb#L13-L15).

Have you confirmed this is still a problem on AMS rc4 and Rails 5 beta2?

In any case, I agree that our test suite needs to add ActionController::API testing!